js/ui/save-html.js

```diff
@@ -75,6 +80,17 @@ define(
                 }
                 ui.freshModal("Save Snapshot", $div);
             }
+        // Create and download an EPUB 3 version of the content
+        // Using (by default) the EPUB 3 conversion service set up at labs.w3.org/epub-generator
+        // For more details on that service, see https://github.com/iherman/respec2epub
+        ,   makeEPubHref: function(){
+                if (!conf.epubGenTool) {
+                    conf.epubGenTool = "https://labs.w3.org/epub-generator/cgi-bin/epub-generator.py";
+                }
+                var epubHref = conf.epubGenTool;
+                epubHref+= "?type=respec&url=" + doc.location.href;
```

Comment:
I am not sure what you mean... What the service needs is a form that can be reused in another GET service right away (the way it works is that it pushes the respec source through the separate spec generator service that Robin set up), and I am not sure that escape would not create problems. That value is just reused by the script right away...